x-pack/metricbeat/modules.d/googlecloud.yml.disabled

```diff
@@ -4,7 +4,9 @@
 - module: googlecloud
   metricsets:
     - compute
+    - storage
   zone: "us-central1-a"
+  region: "us-central1"
```

Comment:
Should compute and storage be in separate config section? Because one is taking zone and the other is taking region as config.

Reply:
I would say that yes 🤔

Reply:
@sayden ^^ WDYT? 😄

Reply:
I overlooked at this the first time but yeap, I think it's better to separate them too. Thanks! :grimacing: